src/reactionsystem.jl

```diff
@@ -1380,6 +1391,7 @@ function Base.convert(::Type{<:SDESystem}, rs::ReactionSystem;
     flatrs = Catalyst.flatten(rs)
     error_if_constraints(SDESystem, flatrs)
 
+    isnothing(noise_scaling) && (noise_scaling = get_noise_scaling(rs)) # Required until passing nosie into SDEProblem can be depricated.
```

Comment:
This needs to use `flatrs` not `rs`.

Reply:
Also, make sure `flattening`, `merging`, and all the other various API functions properly copy over the noise scaling field from system to system. (i.e. `extend`, `compose`, etc) You probably need to go through this file and the network api file carefully to check you haven't missed anywhere. Finally, make sure to update equality testing in the network API file.